scene/3d/sprite_3d.cpp

```diff
@@ -1056,7 +1048,7 @@ void AnimatedSprite3D::_notification(int p_what) {
 			double remaining = get_process_delta_time();
 
 			while (remaining) {
-				double speed = frames->get_animation_speed(animation) * speed_scale;
+				double speed = frames->get_animation_speed(animation) * abs(speed_scale);
```

Comment:
This should use `Math::abs()` instead of `abs()` (same for other occurrences). `abs()` is not as portable and you often get compilation issues with it.

Reply:
Addressed.